10_hourly_supplemental_o2_flat.sql

```diff
@@ -6,7 +6,7 @@ SELECT
 subject_id,
 hadm_id,
 stay_id,
-DATETIME_TRUNC(charttime, HOUR) as charttime,
+DATETIME_TRUNC(charttime, HOUR) as chart_hour,
```

Comment:
Yeah I considered similar names to those, but when I weighed the informational benefit with the convenience cost of typing all of those extra characters, I decided against it, haha

Reply:
perhaps generations later, Edward III, fed up with the bad field names of LifeBell, Inc's MIMIC-VII tabular data will hunt down the origination of 'chart_hour' in the dusty github pull request archives, look to the sky with fists raised, and shout "curse you grandfatherrrr" but it will fall upon deaf ears